### DISCUSSION TITLE
applying waybar themes

### DISCUSSION THREAD

**Discussion by FuckingToasters** [prasanthrangan/hyprdots#593] [Q&A]
hello, so i used following github repo to install hyperland: https://github.com/JaKooLit/Arch-Hyprland/tree/main

It included a few waybar themes, i now found ones not included inside this rep so i copied them over at .config/waybar/style because this is the directonary where the css files i can choose from the repo provided above are located in.

The new themes from this hyprdots repo do show up in the selection list but  when selecting any of the new themes, a standard black theme becomes applied.

Hopefully someone could help me out.

Regards,
FuckingToasters

**Comment by Balssh** [comment]
I am currently testing JaKooLit's dots after being on this for about 4 months.  The implementation of waybar is a bit different there than it is here.
A translation can be done, but it requires a certain amount of effort, are you willing to put that up?

**Comment by kRHYME7** [comment]
Hi! Lets move this to Discussion I guess. 


> The new themes from this hyprdots repo do show up in the selection list but when selecting any of the new themes, a standard black theme becomes applied.


I hope you did not miss this 
``` 
@import "themes/theme.css";

 ```
 
This is a symlink of those custom themes

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/089f165d-82d6-4aa3-a9df-0e80848fbe48)


![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/b6c2a25b-e3a8-4ccf-b924-e5e89ea9cd74)

**Comment by JaKooLit** [comment]
interesting....
